src/Nox.Reference.VatNumbers/Models/ValidationResult.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Nox.Reference.VatNumbers.Models
+{
+    // TODO maybe make record?
+    public class ValidationResult
+    {
+        public ValidationStatus ValidationStatus => ValidationErrors?.Count == 0 ? ValidationStatus.Valid : ValidationStatus.Invalid;
+        public List<string?> ValidationErrors { get; } = new List<string?>();
+
+        public ValidationResult(string? validationError = null)
```

Comment:
I'd recommend to set constructor as private and introduce static methods to create instance CreateEmpty() CreateWithError (static factory)

Reply:
added a parameterless constuctor